File: location_service/LocationService.Core/Domain/Entity/LocationEntity.cs
namespace LocationService.Core.Domain.Entity
{
    public class LocationEntity
    {
        public long Id { get; set; }
        public long CountryId { get; set; }
        public long ProvinceId { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string Detail { get; set; }
        public string Name { get; set; }
    }
}

File: accommodation_service/src/main/java/huy/project/accommodation_service/core/domain/entity/LocationEntity.java
package huy.project.accommodation_service.core.domain.entity;

import lombok.*;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class LocationEntity {
    private Long id;
    private Long countryId;
    private Long provinceId;
    private Double longitude;
    private Double latitude;
    private String detailAddress;
}
